src/app/component-status/component-status.component.ts

```diff
@@ -161,22 +282,29 @@ export class ComponentStatusComponent implements OnInit {
     };
     const url = environment.githubApi + '/projects/columns/' + columnId + '/cards';
     return this.http.get<GithubCard[]>(url, options).pipe(
-      map((cards) => {
-        return cards.map((card) => {
-          let githubIssueNo = 0;
-          const matches = card.content_url.match(/issues\/(\d+)$/);
-          if (matches.length === 2) {
-            githubIssueNo = +matches[1];
-          }
-          return <GithubIssue>{
-            number: githubIssueNo,
-            status,
-          };
-        });
-      })
+      catchError((_) => {
+        this.gitHubError = true;
+        this.isLoading = false;
+        return of([] as GithubCard[]);
```

Comment:
isn't "as GithubCard[]" the React way of casting ? I would use <GithubCard[]>[] instead. It is a very minor thing, so you may ignore it, I am just curious

Reply:
https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#new-tsx-file-extension-and-as-operator